awx/main/migrations/0084_v360_workflow_approval.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 2.2.4 on 2019-08-02 17:51
+
+import awx.main.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0083_v360_job_branch_overrirde'),
```

Comment:
You'll need to rebase again and update this migration to be `0085`, since this recently merged:

https://github.com/ansible/awx/blob/devel/awx/main/migrations/0084_v360_token_description.py